src/api/app/components/notification_notifiable_link_component.rb

```diff
@@ -41,6 +41,10 @@ def notifiable_link_text
       else
         "Removed as #{role} of a project"
       end
+    when 'Event::AddedUserToGroup'
+      "You got added to group '#{@notification.event_payload['group']}'"
```

Comment:
```suggestion
      "#{event_payload['who'] || Someone} added you to the group '#{@notification.event_payload['group']}'"
```

Reply:
Changed

Reply:
This is the result:

![Screenshot 2024-06-12 at 12-59-17 Open Build Service](https://github.com/openSUSE/open-build-service/assets/2581944/8c3446f0-8dfa-4e77-b384-24598715f1ce)